backend/app/ingestion/pdf_processor.py

```
@@ -452,6 +453,57 @@ def process_pdf(
+            pii_field_count += len(pii_matches)
+
+            # Encrypt originals if passphrase available
+            if pii_mode == "encrypt":
+                try:
+                    encrypted = encrypt_pii_fields(pii_matches, pii_passphrase)
+                    pii_vault.append({
+                        "evidence_id": ev["id"],
+                        "fields": encrypted,
+                    })
+                except Exception:
+                    pass  # Encryption failure is non-fatal; text is still redacted
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)